application/controllers/rests.py

```diff
@@ -0,0 +1,92 @@
+from flask_restful import Resource, reqparse, request
+from flask_restful import fields, marshal_with, marshal
+from application.models.rest import Rest
+from application import db
+
+rest_fields = {
```

Comment:
It makes a lot of sense to me to consolidate all of these attributes into one object. Nice work figuring out this data flow with the FE team @Chulstro